### ISSUE TITLE
Accessibility: Plots pane history filmstrip should be a tab group

### ISSUE THREAD

**Issue by jmcphers** [posit-dev/positron#426]
Currently, each plot in the history filmstrip is a separate tabstop. You can select a plot by tabbing until it is selected, and then pressing Space or Enter. Shift+tab, naturally, moves backwards.

<img width="112" alt="image" src="https://user-images.githubusercontent.com/470418/231566172-3c395472-a5e7-4e5a-8705-dee4b2f6b0a6.png">

This works fine for simple cases (where you have, say, 10 plots or fewer), but as the number of plots is practically unbounded, it can be difficult to tab past the Plots pane once the history has accumulated enough items. For accessibility purposes, the entire history should be a single tab group so that it can be tabbed over as the user moves through the top level UI items; the user can either send focus into the group or use arrows to navigate inside it in order to select and view plots.

**Comment by jonvanausdeln** [comment]
### Verified Fixed
Positron Version(s) : [2025.05.0-58](https://github.com/posit-dev/positron-builds/releases/tag/2025.05.0-58)
OS Version(s) : Windows 11

### Test scenario(s) 

Created 20 plots and arrow and tab navigation work as expected
